### ISSUE TITLE
Add NlpBackend to site

### ISSUE THREAD

**Issue by haarp** [mozilla/ichnaea#405]
There's an "app" not mentioned on the MLS site yet (it's actually more of a plugin). It's a backend for the UnifiedNlp framework that provides location services for Android without Google.

https://github.com/microg/IchnaeaNlpBackend

It should probably be listed under the "Apps" section, me thinks.

**Comment by Djfe** [comment]
Yeah that app should be mentioned there I think, at least it would make sense in my opinion

maybe this app could be mentioned as well, because it allows using the offline MLS cell database
https://github.com/n76/Local-GSM-Backend

**Comment by hannosch** [comment]
I've created a new wiki page to list related software projects, so I'm not the bottleneck for adding something. The webpage is currently tied to the rest of software, which makes it hard to change it. I've added a new link to the apps page to point to https://wiki.mozilla.org/CloudServices/Location/Software for community projects. That will only be rolled out in the next two weeks.

Can one of you add the projects to https://wiki.mozilla.org/CloudServices/Location/Software?
